integration-tests/ccip-tests/actions/ccip_helpers.go

```diff
@@ -1574,6 +1575,82 @@ func (lane *CCIPLane) AddToSentReqs(txHash common.Hash, reqStat *testreporters.R
 	return rcpt, nil
 }
 
+func (lane *CCIPLane) MultiSend(noOfRequests int, msgType string, multiSendAddr common.Address) error {
+	var ccipMultSend []contracts.CCIPMsgData
+	var tokenAndAmounts []router.ClientEVMTokenAmount
```

Comment:
Hmm, in case `msgType` is not `TokenTransfer`, this slice is `nil`, and since it goes inside a msg struct, encoding/decoding it may show some weird behaviors and hard-to-debug bugs.
I'd suggest you ensure it's always initialized:
```suggestion
	tokenAndAmounts := []router.ClientEVMTokenAmount{}
```